docker-compose.dev.yml

```diff
@@ -4,9 +4,13 @@ services:
   identifier:
     ports:
       - "80:80"
-  db:
+  triplestore:
     ports:
       - "8890:8890"
+  mocklogin:
```

Comment:
This means we need to add it to the deployed apps docker.overrides.yml file, right? I guess that's how it is done in the other apps?

Reply:
yes indeed, it has to be added on the server where it will be deployed also